package.json

```diff
@@ -88,7 +88,7 @@
     "typescript": "~4.9.5",
     "yargs": "^17.7.2"
   },
-  "packageManager": "yarn@3.3.0",
+  "packageManager": "yarn@4.2.2",
```

Comment:
This is the latest version of Yarn. It's further than what the module template has, but I will submit a PR to upgrade it there.